src/utils/RealtimeFeedUtils.js

```diff
@@ -172,8 +87,33 @@ function getDelayInformation(stopID: String, tripID: String, rtf: Object): ?Obje
   };
 }
 
+function getVehicleInformation(
+  routeID: String,
+  tripID: String,
+  vehicles: Object,
+): ?Array<Object> {
```

Comment:
also not sure how the data is formatted but should the return type here by `?Array<Object>` or `?Object`?

Reply:
ohh u right